core/modules/compare/helpers/notifications.ts

```diff
@@ -0,0 +1,21 @@
+import i18n from '@vue-storefront/i18n'
```

Comment:
I'm strongly against this approach. Action should throw an exception when something is wrong or return something or even nothing when an operation was finished. Notification part is purely on theme responsibility. The theme can have different notifications mechanisms, different approaches or event wants to not show notifications. In most cases, they also want to change notification message. So basically we shouldn't have notifications in actions for better flexibility.

Reply:
@patzick yeah, completely agree I even know about that. I implemented that way because I didn't want to introducing a different way of handling notification. I think it'a a separated task and probably it could cause some problems with upgrading etc.. Soo basically I can do this (return something or raise error + handle notification in the theme), but the question is whether won't it cause some BC-breaks?